sdk/communication/communication-common/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.6 (Unreleased)
+## 1.0.0-beta.6 (2021-03-09)
```

Comment:
You will have to add something so just use update '<package-name>' version... look for examples below in the file